src/Resources/config/services/services.yml

```diff
@@ -4,8 +4,9 @@ services:
         autowire: true
         public: false
 
-    EzSystems\EzPlatformGraphQL\Command\:
-        resource: '../../Command/*'
+    EzSystems\EzPlatformGraphQL\Command\GeneratePlatformSchemaCommand:
+        arguments:
+            $schemaRootDir: '%ezplatform.graphql.schema.root_dir%'
```

Comment:
Side:
Is it just me or is this parameter [set](https://github.com/ezsystems/ezplatform-graphql/blob/v1.0.1/src/DependencyInjection/EzSystemsEzPlatformGraphQLExtension.php#L86) [twice](https://github.com/ezsystems/ezplatform-graphql/blob/v1.0.1/src/DependencyInjection/Compiler/RegisterSchemaDirectoryParametersPass.php#L28)? :)

Reply:
Hummm, indeed. Leftovers from the initial implementation I guess. I'll get rid of the compiler pass, I think.